examples/config_file/src/main/java/io/github/outscale/example/App.java

```diff
@@ -22,9 +22,24 @@ public static void createConfigurationFile(String path) {
     String accessKey = System.getenv("OSC_ACCESS_KEY");
     String secretKey = System.getenv("OSC_SECRET_KEY");
     String region = System.getenv("OSC_REGION");
+    String endpoint_api = System.getenv("OSC_ENDPOINT_API_NOPROTO");
```

Comment:
The naming convention is CamelCase in Java therefore it is better to use `endpointApi`